gui.py

```diff
@@ -234,10 +223,11 @@ def halt_simulation(self):
 
 
     def execute_step(self):
```

Comment:
The Step Execution button is not working on your gui. You fixed the indicator for the current register that the Sim is on, but we need the button execute the opcode and then move to the next instruction